README.md

```diff
@@ -79,6 +79,7 @@ Or with [Faros Community Edition](https://github.com/faros-ai/faros-community-ed
 | --src-docker-options "\<string\>" |          | Set additional options to pass to the `docker run <src>` command                                  |
 | --dst-docker-options "\<string\>" |          | Set additional options to pass to the `docker run <dst>` command                                  |
 | --debug                           |          | Enable debug logging                                                                              |
+| --debug-secrets                   |          | Enable debug logging with visible secrets                                                         |
```

Comment:
Let's not add a separate flag and instead just always redact when --debug is provided.